app/models/operations/rate_builder.rb

```diff
@@ -103,10 +103,10 @@ def build_premium_tables
         ).each do |product|
           product.premium_tables << ::Products::PremiumTable.new(
             effective_period: applicable_range,
-            rating_area_id:,
+            rating_area_id: rating_area_id,
             premium_tuples: tuples
           )
-          product.premium_ages = tuples.map(&:age).minmax
+          product.premium_ages = Range.new(tuples.map(&:age).min, tuples.map(&:age).max)
```

Comment:
Thanks for including these changes. They will fix the nil premium_ages issue on application start, which occurred after the Mongoid version upgrade.